agir/people/models.py

```diff
@@ -659,6 +660,14 @@ def has_perm(self, perm, obj=None):
         """Simple raccourci pour vérifier les permissions"""
         return self.role.has_perm(perm, obj)
 
+    @property
+    def number_of_groups_animated(self):
+        anime = 0
+        for g in SupportGroup.objects.all():
+            if self in g.managers:
+                anime += 1
+        return anime
```

Comment:
C'est probablement plus rapide de faire directement le compte dans la requête, au lieu de faire un boucle :
```suggestion
    @property
    def number_of_groups_animated(self):
        return SupportGroup.objects.filter(
            memberships__person=self,
            memberships__membership_type__gte=Membership.MEMBERSHIP_TYPE_MANAGER,
        ).count()
```
(il faudra juste ajouter l'import de `Membership`)